content/en/user-guide/aws/elastic-kubernetes-service/index.md

```diff
@@ -328,24 +338,32 @@ $ awslocal eks list-clusters
 }
 {{< / command >}}
 
-Simply configure your Kubernetes client (e.g., `kubectl` or other SDK) to point to the `endpoint` specified in the `create-cluster` output above. Depending on whether you're calling the Kubernetes API from the local machine or from within a Lambda, you may have to use different endpoint URLs (`https://localhost:6443` vs `https://172.17.0.1:6443`).
+To interact with your Kubernetes cluster, configure your Kubernetes client (such as `kubectl` or other SDKs) to point to the `endpoint` provided in the `create-cluster` output mentioned earlier. However, depending on whether you're calling the Kubernetes API from your local machine or from within a Lambda function, you might need to use different endpoint URLs.
+
+For local machine interactions, use `https://localhost:6443` as the endpoint URL. If you are accessing the Kubernetes API from within a Lambda function, you should use `https://172.17.0.1:6443` as the endpoint URL.
```

Comment:
Maybe better use `host.docker.internal` instead of the hardcoded IP address, or at least mention something like "... (assuming that `172.17.0.1` is the IP address of the Docker network bridge)".